java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1052,4 +1052,34 @@ public void pfcount(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.pfcount(new String[] {"foo"}).get());
         assertTrue(executionException.getCause() instanceof RequestException);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void pfmerge(BaseClient client) {
+        String key1 = "{test}-hll1-" + UUID.randomUUID();
+        String key2 = "{test}-hll2-" + UUID.randomUUID();
+        String key3 = "{test}-hll3-" + UUID.randomUUID();
+        assertEquals(1, client.pfadd(key1, new String[] {"a", "b", "c"}).get());
+        assertEquals(1, client.pfadd(key2, new String[] {"b", "c", "d"}).get());
+        // new HLL
+        assertEquals(OK, client.pfmerge(key3, new String[] {key1, key2}).get());
+        assertEquals(
+                client.pfcount(new String[] {key1, key2}).get(), client.pfcount(new String[] {key3}).get());
+        // existing HLL
+        assertEquals(OK, client.pfmerge(key1, new String[] {key2}).get());
+        assertEquals(
+                client.pfcount(new String[] {key1, key2}).get(), client.pfcount(new String[] {key1}).get());
```

Comment:
Is there no way to check the actual contents of the merged HyperLogLog value?

Reply:
If it exists - is not documented: https://redis.io/commands/?group=hyperloglog
It is possible to make a regular `get` request for HLL, but I don't think that it is an expected way to work with it. The data in HLL is encoded, it is not a plain-text struct.